tests/app/main/views/test_suppliers.py

```diff
@@ -191,7 +191,7 @@ def test_should_show_user_details_on_page(self, data_api_client):
         assert response.status_code == 200
         assert "Test User" in response.get_data(as_text=True)
         assert "test.user@sme.com" in response.get_data(as_text=True)
-        assert "10:33:53" in response.get_data(as_text=True)
+        assert "09:33:53" in response.get_data(as_text=True)
```

Comment:
Is there a time zone in there now, if so can our assertion cover that being in there?

Reply:
There's no timezone added in this instance.